icon-registry/icons-static/cypress-accessibility-outline_x16.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M15 7.99927C15 11.8653 11.866 14.9993 8 14.9993C4.13401 14.9993 1 11.8653 1 7.99927C1 4.13327 4.13401 0.999268 8 0.999268C11.866 0.999268 15 4.13327 15 7.99927Z" fill="currentColor" class="icon-light"/>
```

Comment:
Can we have the default fill as transparent to avoid showing a black circle by default?
```suggestion
  <path d="M15 7.99927C15 11.8653 11.866 14.9993 8 14.9993C4.13401 14.9993 1 11.8653 1 7.99927C1 4.13327 4.13401 0.999268 8 0.999268C11.866 0.999268 15 4.13327 15 7.99927Z" fill="transparent" class="icon-light"/>
```

Reply:
Thanks @elevatebart 

I updated this.